opentelekomcloud/resource_opentelekomcloud_identity_project_v3.go

```diff
@@ -47,6 +47,16 @@ func resourceIdentityProjectV3() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
+
```

Comment:
why add enabled, and is_domain as you delete the check assert of them

Reply:
Because it will show in project resource response after create a project